appveyor.yml

```diff
@@ -21,25 +21,28 @@ matrix:
       MSYSTEM: MSVC
 
 for:
-  -
-    matrix:
+  - matrix:
       only:
         - MSYSTEM: MINGW64
+    install:
+      ps: "C:\\msys64\\usr\\bin\\bash -lc \"pacman -S --quiet --noconfirm --needed re2c 2>&1\""
     build_script:
       ps: "C:\\msys64\\usr\\bin\\bash -lc @\"\n
-      pacman -S --quiet --noconfirm --needed re2c 2>&1\n
+      ./misc/ci.py 2>&1\n
```

Comment:
It actually would be best to run this script only once. It shouldn't find anything new on different platforms.

Reply:
misc/ci.py is part of the ninja source code, it should be exercised on all platforms where it's easy to do so.

You're correct that if it fails on one platform, it should fail on all platforms, but there are, or might be in the future, downstream forks of Ninja which may not use the same CI targets that Ninja does. (e.g., may only configure a Travis CI account, or an AppVeyor account, but not both).

This change ensures that misc/ci.py is always working, even for downstream projects that only use a subset of the CI configurations.